### DISCUSSION TITLE
Please help me to collect nice musics

### DISCUSSION THREAD

**Discussion by p1scescom** [NAIST-SE/AttractingContributors#13] [General]
https://github.com/p1scescom/AttractingContributors-Music/

**Comment by yuka1369** [comment]
I add my favorite music.and I request pull request.
But I'm beginner.Please tell me if I make a mistake.

**Comment by satokibi44** [comment]
I've already added!

**Comment by eedy8888** [comment]
added please help me
https://github.com/eedy8888/AttractingContributors

**Comment by 278Mt** [comment]
added! and plz help my favorite movies!
https://github.com/278mt/AttractingContributors
thx;-)

**Comment by manzoku23** [comment]
Added my favorite one.
Please help me
https://github.com/Exe-dev/AttractingContributors-POKEMON

**Comment by sergiodl-naist** [comment]
Added one that I always have near my heart
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers
